apkleaks/apkleaks.py

```diff
@@ -94,6 +94,7 @@ def decompile(self):
 		except Exception:
 			pass
 		comm = "%s" % (" ".join(quote(arg) for arg in args))
+		comm = comm.replace("\'","\"")
```

Comment:
This. Should this replace everything? Isn't the problem not being able to read the `self.file` _(line 91)_?